torchvision/ops/boxes.py

```diff
@@ -133,6 +134,54 @@ def clip_boxes_to_image(boxes: Tensor, size: Tuple[int, int]) -> Tensor:
     return clipped_boxes.reshape(boxes.shape)
 
 
+def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
+    """
+    Converts boxes from given in_fmt to out_fmt.
+
```

Comment:
Can we specify in the documentation string what are the possible supported formats, as well as what they represent?
Something like:
```
Supported formats are:
    'xyxy': boxes are represented via their corners, with x1, y1 the top left corner and ...
```

Reply:
added 👍